src/wombats/sockets/game.clj

```diff
@@ -212,7 +211,9 @@
       (let [formatted-message {:username github-username
                                :message message
                                :color (get-player-color game-id chan-id)
-                               :timestamp (str (l/local-now))}]
+                               :timestamp (->> (t/now)
```

Comment:
Agreed, I think we should talk about a reusable lib for the Client & API for these type of `fn`s. Going to merge for now and create a ticket for a util lib. Also might be nice for using in the lambda fn.